### ISSUE TITLE
Altered app bundle fingerprint in Nightly

### ISSUE THREAD

**Issue by marchare555** [renpy/renpy#4536]
Me again! This time I'm reporting the following error in Ren'Py 7.6.0.23041402:

I transferred the android.keystore and bundle.keystore files from 7.5.3 (/rapt folder) into the game's root folder in Ren'py 7.6 and I am now getting this error from Google Play Console:
```
Your Android App Bundle is signed with the wrong key. Ensure that your App Bundle is signed with the correct signing key and try again. Your App Bundle is expected to be signed with the certificate with fingerprint:
```

If I export a playstore build from 7.5.3, I do not get this error. I also tried setting the "update_keys " variable in android.json to false after reading the nightly android doc but same result.

**Comment by renpytom** [comment]
Can you check the local.properties files and bundle.properties files in the new instance of Ren'Py's project/ directory, and see if the paths are correct? I just tried this here, and I was able to run through the sequence you gave and wind up with a signed and working project.

**Comment by marchare555** [comment]
Here's what I get in the local.properties and bundle.properties files:
```
key.alias=android
key.store.password=android
key.alias.password=android
key.store=D:/renpy-7-nightly-2023-04-16-7.6.0.23041507+nightly-sdk/TheSymbiant_1.2/android.keystore
sdk.dir=D:/renpy-7-nightly-2023-04-16-7.6.0.23041507+nightly-sdk/rapt/Sdk
```

Here's what I have in the game's root folder:
![image](https://user-images.githubusercontent.com/34620611/232308218-2911abab-c3ba-41f4-97c9-409239ae6756.png)

I tried again today but I still get a different fingerprint.

Also worth noting: local.properties and bundle.properties only get created **after** a key is generated or after a build is exported. It may be useful to mention this in the documentation also, to avoid possible confusion for new android devs.

Also question: is there an easy way to view the fingerprint aside from exporting + uploading to Google Play? I tried the "keytool" command from the Java installation folder but it asks for a passphrase.
